README.md

```diff
@@ -41,13 +41,21 @@ whole log file as an attachment.
 What it sets up
 ---------------
 
+* [Alfred] for searching for files and applications
 * [Bundler] for managing Ruby libraries
+* [Dash] for API documentation management
 * [Exuberant Ctags] for indexing files for vim tab completion
+* [Firefox]
 * [Foreman] for managing web processes
-* [gh] for interacting with the GitHub API
```

Comment:
We might not want to bother making this change, since after https://github.com/thoughtbot/laptop/pull/384 is merged, it will fix itself once we rebase. Or we can just integrate the changes from that PR directly (though it will result in more of a headache during the rebase).